whois-update/src/main/java/net/ripe/db/whois/update/domain/UpdateMessages.java

```diff
@@ -176,6 +176,10 @@ public static Message invalidMaintainerForOrganisationType(final CharSequence or
         return new Message(Type.ERROR, "Value '%s' can only be set by the RIPE NCC for this organisation.", orgType);
     }
 
+    public static Message invalidMaintainerName() {
+        return new Message(Type.ERROR, "Maintainer name must end with a -MNT suffix");
```

Comment:
Update the message to clarify that the error is on CREATE only:
`When creating a MNTNER the name must end with an -MNT suffix`

Reply:
done